insights/client/config.py

```diff
@@ -232,6 +232,12 @@ def _core_collect_default():
         'action': 'store_true',
         'group': 'debug'
     },
+    'module': {
```

Comment:
Can we make this option hidden from the `--help` output by default, since it's not really meant for public consumption?